issue-stats.py

```diff
@@ -9,10 +9,18 @@
 
 
 REPOS = [
-  'bazelbuild/starlark',
-  'bazelbuild/bazel-website',
-  'bazelbuild/bazel-skylib',
+  'bazelbuild/apple_support',
   'bazelbuild/bazel',
+  'bazelbuild/bazel-blog',
+  'bazelbuild/bazel-federation',
+  'bazelbuild/bazel-skylib',
+  'bazelbuild/bazel-toolchains',
```

Comment:
bazel-toolchains, skydoc, codelabs, apple_support and starlark all have pretty clear owners. Should we triage their issues too? If not, can we turn this into a flag instead?

Reply:
They do have clear owners. I want to get then into the latency metrics dashboard. That is important for a product management view.

Reply:
SGTM